resources/keb/files/swagger.yaml

```diff
@@ -1354,12 +1225,18 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+        '500':
+          description: Internal server error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
```

Comment:
I'm not 100% sure if we should use the generic `Error` or create an appropriate error message for each code. For now I decided to use the generic `Error` as we have the same approach in the `Instances` section.